src/main/scala/where/Condition.scala

```diff
@@ -76,3 +76,10 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
+  def toQuery(identifiableMap: IdentifiableMap): String =
+    "NOT " + node.toQuery(identifiableMap) + "-[]-()"
```

Comment:
`node.toQuery(...)` will draw out extra properties of a node if you pass a `CypherNode`, but it looks like you just want the identifier here. look at `ActionItem.scala#48`, you should instead grab **just** the identifier and wrap it in parens. you should also throw `IdentifierDoesntExistException` if it's not found